{{cookiecutter.project_slug}}/README.md

```diff
@@ -17,12 +17,17 @@ To activate the environment run:
 To update this environment with your production dependencies run:
 
     conda env update --file=environment.yml
-{% else %}
+{% elif cookiecutter.package_manager == 'pip' %}
+To set up your local development environment, please use a fresh virtual environment.
+
 Then run:
 
     pip install -r requirements.txt -r requirements-dev.txt
+{% else %}
+To set up your local development environment, run:
+    poetry install
 {% endif %}
-You can now {% if cookiecutter.create_cli == 'yes' %}run the module from the `src` directory with `python -m {{ cookiecutter.module_name }}`{% else %}import functions and classes from the module with `import {{ cookiecutter.module_name }}`{% endif %}.
+You can now {% if cookiecutter.create_cli == 'yes' %}run the module from the `src` directory with `{{ 'poetry run ' if cookiecutter.package_manager == 'poetry' else '' }}python -m {{ cookiecutter.module_name }}`{% else %}import functions and classes from the module with `import {{ cookiecutter.module_name }}`{% endif %}.
```

Comment:
this is some next level jinja template magic right here 😅
the price we pay by providing so many options I guess...

Did you check if this renders correctly for all combinations of CLI and poetry?

Reply:
Haha, yes I felt a bit bad about all that complexity, but I did check and it works for all combinations. Maybe I should add a couple of tests to verify that though.

Alternatively, we could simplify this at the cost of a bit of duplication by putting this section inside the `{% if cookiecutter.package_manager == ...` conditional above.